stable/mongodb-replicaset/values.yaml

```diff
@@ -29,9 +29,10 @@ image:
 
 # Specs for busybox image
 
-initContainers:
+copyConfigImage:
   busybox:
     repository: busybox
+    tag: latest
```

Comment:
Stable version, please. Otherwise, we wouldn't have to do that.

Reply:
Fixed